code/ZAS/XGM_gases.dm

```diff
@@ -128,7 +128,7 @@
 /datum/gas/cryotheum
 	id = GAS_CRYOTHEUM
 	name = "Cryotheum"
-	short_name = "O<sub>2</sub>C"
+	short_name = "O<sub>2</sub>β"
```

Comment:
I'd change it to Theta (θ), since theum/theta are similar sounding.

Reply:
I don't understand why that matters.

Reply:
I just think it'd be nicer.